ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/security/auth.guard.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, Router } from '@angular/router';
+import { OidcSecurityService } from 'angular-auth-oidc-client';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { PageUrls } from 'src/app/modules/shared/constants/page-url.constants';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthGuard implements CanActivate {
+  constructor(private oidcSecurityService: OidcSecurityService, private router: Router) {}
+
+  canActivate(): Observable<boolean> {
```

Comment:
Added a guard but not been applied anywhere?

Reply:
It's not required (was following Admin Web), will remove it. As we have individual and representative guard present which covers for service web as before.